packages/account/src/Sections/Profile/PersonalDetails/personal-details.jsx

```diff
@@ -922,7 +922,12 @@ export const PersonalDetailsForm = ({
                                                     </fieldset>
                                                 )}
                                                 {'tax_identification_number' in values && (
-                                                    <fieldset className='account-form__fieldset'>
+                                                    <fieldset
+                                                        className={classNames('account-form__fieldset', {
+                                                            'account-form__fieldset--tin':
```

Comment:
change the modifier name to something else for readability ... because by reading tin we won't know whats happening..

Reply:
I think it's ok, we use tin instead of tax_identification_number quite a lot